book/website/community-handbook/contributing.md

```diff
@@ -1,11 +1,20 @@
 (ch-contributing)=
-# Contributing and Developing Chapters - Templates and Workflow
+# Chapter Templates and Contribution Workflow
 
 As _The Turing Way_ project grows and evolves, we actively welcome and encourage continuous contributions from community members in the form of new chapters and case studies. 
 However, these new contributions need to be consistent with the overall theme, purpose, format, and style of the book's existing content.
 
 Since _The Turing Way_ is written asynchronously by multiple authors around the world, we created a series of templates to guide different kinds of content development while maintaining a consistent format of the book.
 
+```{figure} ../figures/templates.png
```

Comment:
I think this edit should be in a separate pull request/issue, but I am not sure.